### DISCUSSION TITLE
Option to disable toggle button for side menu and force always open

### DISCUSSION THREAD

**Discussion by muscanto** [snipe/snipe-it#9190] [Ideas]
Small and simple request -

Could there be an option to disable the toggle button for expanding the side menu?  We want it always expanded.

I looked into modifying the page files but it's a little more than I want to take on and removing the call for the toggle forces it into collapsed mode.

**Comment by snipe** [comment]
There is a cookie set when you open/close the menu that keeps it in whatever state you had it in last. Is that not sufficient?

**Reply by muscanto** [reply]
It only works about half the time when your browser actually remembers your settings.  I'm constantly hitting that toggle.  Honestly, I have no need for the minimized menu ever.

**Reply by snipe** [reply]
I have never encountered a time when that setting was not remembered in the same browser session.

**Reply by muscanto** [reply]
Sure, but I will open and close the browser multiple times a day and often times hop on another computer.  I'd just like to be able to disable it so the menu stays open all the time.
